db/migrate/20150707165322_create_tasks.rb

```diff
@@ -0,0 +1,10 @@
+class CreateTasks < ActiveRecord::Migration
+  def change
+    create_table :tasks do |t|
+      t.text :content
+      
+      t.integer :property_id
```

Comment:
I'm unsure if we'll need the property_id.

Primarily because I can't imagine us ever needing to do the reverse operation:
i.e., we have a task, and we want to find its property.